### DISCUSSION TITLE
What do the abbreviations mean?

### DISCUSSION THREAD

**Discussion by yatakoi** [kiali/kiali#6251] [General]
Could you tell me what the abbreviations mean СDS, LDS, RDS, ECDS? Where you can read about them
```
[root@m1 temp]# istioctl proxy-status
NAME                                 CLUSTER        CDS          LDS        EDS          RDS          ECDS         ISTIOD                     VERSION
echo-v1-768f8b6fc7-h28kw.default     Kubernetes     NOT SENT     SYNCED     NOT SENT     NOT SENT     NOT SENT     istiod-57c965889-bh5xs     1.17.2
echo-v2-787b774f49-w5zzm.default     Kubernetes     NOT SENT     SYNCED     NOT SENT     NOT SENT     NOT SENT     istiod-57c965889-bh5xs     1.17.2
```

Thank you.

**Comment by hhovsepy** [comment]
I found this information while searching, it is taken from book "Istio In Action":
CDS: Cluster Discovery Service
LDS: Listener Discovery Service
EDS: Endpoint Discovery Service
RDS: Route Discovery Service
ECDS: Envoy Cluster Discovery Service

Some blogpost including it:
https://medium.com/airbnb-engineering/improving-istio-propagation-delay-d4da9b5b9f90

**Reply by jmazzitelli** [reply]
Just a side note - this is more of a question for the main Istio team, not the Kiali team. Those are related to the envoy proxy and nothing to do with Kiali.

**Comment by yatakoi** [comment]
Thank you!
